reflect.go

```diff
@@ -890,6 +891,23 @@ func (r *Reflector) typeName(t reflect.Type) string {
 	return t.Name()
 }
 
+// Only split if the rune after the comma is a letter
+// This way, we prevent splitting regexes
+func splitOnComma(tagString string) []string {
+	tags := make([]string, 0)
+	var lastRune rune = 0
+	lastSplit := 0
+
+	for i, chr := range tagString {
+		if lastRune == ',' && unicode.IsLetter(chr) {
```

Comment:
I have doubts about this approach. There is no guarantee that the next character after a comma is the separator. Have you tried using an escape character? i.e. `\`. You could then split on an regexp instead, with a pattern like `[^\\],`.